src/SonarScanner.MSBuild.PreProcessor/Interfaces/IDownloader.cs

```diff
@@ -35,22 +35,22 @@ public interface IDownloader : IDisposable
         /// </summary>
         /// <returns>False if the url does not exist, true if the contents were downloaded successfully.
         /// Exceptions are thrown for other web failures.</returns>
-        Task<Tuple<bool, string>> TryDownloadIfExists(Uri url, bool logPermissionDenied = false);
+        Task<Tuple<bool, string>> TryDownloadIfExists(string url, bool logPermissionDenied = false);
 
         /// <summary>
         /// Attempts to download the specified file.
         /// </summary>
+        /// <param name="url"></param>
```

Comment:
I would drop the comments. I think they do not bring more insights about the method than the method signature.